stellarphot/core.py

```diff
@@ -1163,7 +1170,12 @@ def passband_columns(self, passbands=None):
             passbands = catalog_passbands
         input_passbands = set(passbands)
         missing_passbands = input_passbands - catalog_passbands
-        if missing_passbands:
+        if missing_passbands == input_passbands:
+            # The user only request transformed passbands, give them all of them
```

Comment:
If I interpret the code correctly, if `missing_passbands == input_passbands` then `catalog_passbands` and `input_passbands` have no passbands in common, so what is the logic in replacing `input_passbands` with `catalog_passbands`?  Presumably the user has requested passbands NOT in the catalog in this case, are we not honoring that?

The comment is grammatically unclear which is part of the problem.